apps/src/utils.js

```diff
@@ -235,7 +235,13 @@ export function isInfiniteRecursionError(err) {
 export function browserSupportsCssMedia() {
   var styleSheets = document.styleSheets;
   for (var i = 0; i < styleSheets.length; i++) {
-    var rules = styleSheets[i].cssRules || styleSheets[i].rules;
+    var rules;
+    try {
+      rules = styleSheets[i].cssRules || styleSheets[i].rules;
```

Comment:
Interesting... I think the question this function is asking is, "Does this browser support JS access to CSS media rules?"  It's trying to answer that question by iterating through all loaded stylesheets and accessing them.

Right now if _any_ stylesheet is inaccessible the answer is "No, not supported."  Maybe it would be better to answer yes if any stylesheet is accessible.